transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/XMLPage.java

```diff
@@ -331,7 +331,7 @@ private String selectResourceFromWorkspace(Shell shell, final String extension)
         }
         ClasspathResourceSelectionDialog dialog = null;
         HashSet<String> extensions = new HashSet<>();
-        extensions.add("xml"); //$NON-NLS-1$
+        if (!"xsd".equals(extension)) extensions.add("xml"); //$NON-NLS-1$
```

Comment:
IMO, these are true constants, not "magic" values that would benefit from a defined constant in case they changed.  Adding constants for these just adds extra code with no real benefit to maintenance, code size, etc.